discovery-provider/plugins/notifications/src/email/notifications/index.ts

```diff
@@ -42,11 +40,46 @@ const Results = Object.freeze({
 export const getUsersCanNotify = async (
   identityDb: Knex,
   frequency: EmailFrequency,
-  startOffset: moment.Moment
-): Promise<EmailUsers> => {
-  // const validLastEmailOffset = startOffset.subtract(2, 'hours')
-  const userRows: { blockchainUserId: number; email: string }[] =
-    await identityDb
+  startOffset: moment.Moment,
+  pageCount: number,
+  onPage: (emailUsers: EmailUsers) => Promise<void>,
+) => {
+  let lastUser: number = 0;
+  const maxUser = await identityDb('NotificationEmails').max('userId').where('emailFrequency', '=', frequency)
+  const totalCurrentUsers = parseInt(maxUser[0].max as string)
```

Comment:
wondering if we should rename to something like `maxUserId` instead since total current users isnt entirely accurate

Reply:
ah yeah good idea